contribute/developer_guide/README.md

```diff
@@ -2,14 +2,14 @@
 
 The artefacts associated with a chaos-experiment are summarized below: 
 
-- Submitted in the litmuschaos/litmus repository, under the experiments/*chaos category* folder
+- Submitted in the litmuschaos/litmus repository, under the experiments/*chaos-category* folder i.e, `experiments/generic`
 
   - Experiment business logic (either as ansible playbook, or other). May involve creation of new or reuse of existing chaoslib
-  - Experiment Kubernetes job (executes the business logic, also called litmusbook)
+  - Experiment Kubernetes job (pass the experiment-specific chaos parameters, executes the business logic, also called litmusbook)
```

Comment:
*passes
unclear if the job or the business logic is called litmusbook :D in the latter case please add it to above point

Reply:
also, not sure if litmusbook is still an adequate name for it, as I understand this came from ansible playbook, but not only ansible is used now